.storybook/main.js

```diff
@@ -1,11 +1,13 @@
 const path = require('path');
 
 module.exports = {
-  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  stories: ['../src/**/*.stories.@(js|jsx|ts|tsx)', '../src/**/*.stories.mdx'],
```

Comment:
You can combine the two selectors in to one

```suggestion
  stories: ['../src/**/*.stories.@(js|jsx|ts|tsx|mdx)'],
```